tests/test_bench.py

```diff
@@ -1059,9 +1059,13 @@ def test_count_qubit_numbers_per_compiler(compiler: str, expected_val: list[int]
 
 
 def test_calc_supermarq_features() -> None:
-    qc = get_benchmark("dj", 1, 5)
-    features = utils.calc_supermarq_features(qc)
-    assert type(features) == utils.SupermarqFeatures
+    qcs = [
+        get_benchmark(benchmark, 1, 5)
+        for benchmark in ["dj", "ghz", "random", "qaoa", "vqe", "qnn", "qpeexact", "qpeinexact", "wstate"]
+    ]
+    for qc in qcs:
+        features = utils.calc_supermarq_features(qc)
+        assert type(features) == utils.SupermarqFeatures
```

Comment:
This is kind of guaranteed by static type checking (mypy) already, so this is kind of a useless assertion.
Can't we properly test something here? For example, that a GHZ state has `parallelism=0`, etc. (preferably in separate tests). For the best tests, we would probably have three scenarios per feature:
- lower extreme (f == 0.)
- higher extreme (f == 1.)
- something in-between (0 < f < 1)

If this is too annoying, then feel free to resolve this and keep the (admittedly suboptimal) tests.